FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -836,6 +838,16 @@ namespace edm {
         gInterpreter->SetClassAutoloading(1);
       }
 
+      // Enable/disable automatic parsing of headers
+      //gInterpreter->SetClassAutoparsing(autoClassParser_);
+      if (not autoClassParser_) {
+        // Disable automatic parsing of headers during module construction
+        iReg.watchPreModuleConstruction(
```

Comment:
As we now construct modules at the same time as we are constructing the Source, how does this interact?